ui/tx/TxAssetFlows.tsx

```diff
@@ -0,0 +1,122 @@
+import { Table, Thead, Tbody, Tr, Th, TableContainer, Box, Skeleton, Text, Show, Hide, Divider } from '@chakra-ui/react';
+import _ from 'lodash';
+import React, { useState } from 'react';
+
+import type { PaginationParams } from 'ui/shared/pagination/types';
+
+import useApiQuery from 'lib/api/useApiQuery';
+import { NOVES_TRANSLATE } from 'stubs/noves/novesTranslate';
+import ActionBar from 'ui/shared/ActionBar';
+import DataListDisplay from 'ui/shared/DataListDisplay';
+import AddressEntity from 'ui/shared/entities/address/AddressEntity';
+import Pagination from 'ui/shared/pagination/Pagination';
+
+import TxAssetFlowsListItem from './assetFlows/TxAssetFlowsListItem';
+import TxAssetFlowsTableItem from './assetFlows/TxAssetFlowsTableItem';
+import { generateFlowViewData } from './assetFlows/utils/generateFlowViewData';
+
+interface FlowViewProps {
+  hash: string;
+}
+
+export default function TxAssetFlows(props: FlowViewProps) {
+
+  const { data: queryData, isPlaceholderData, isError } = useApiQuery('noves_transaction', {
+    pathParams: { hash: props.hash },
+    queryOptions: {
+      enabled: Boolean(props.hash),
+      placeholderData: NOVES_TRANSLATE,
+    },
+  });
+
+  const [ page, setPage ] = useState<number>(1);
+
+  const ViewData = queryData ? generateFlowViewData(queryData) : [];
+  const chunkedViewData = _.chunk(ViewData, 10);
```

Comment:
If pagination is not available for this resource on the server side, I question whether we should implement it on the client side.

By the way, our standard page size is 50.

Reply:
I understand the page size is 50 on your side, but on our side we're more comfortable sticking to the original 10 (for our API call).

Reply:
We extract all the "transfers" items in this tab from `api/v2/proxy/noves-fi/transactions/${th_hash}` so it doesn't need pagination but it can have a lot of transfers, for example https://goerli.etherscan.io/tx/0x3fd8edc3cc1c891d2a07d31f20c45561d942f7b7d3487b6f4c708dc9bd0c6d4e

So i chunk the transfers to display 10 per "page"

Reply:
Can we set the page size to 50 then? So it will be consistent with the other lists.

Reply:
Done